docker-assets/initialize-httpd-auth.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+HTTPD_AUTH_CONFIG_DIR=/etc/httpd/auth-conf.d
+AUTH_CONFIG_FILE="${HTTPD_AUTH_CONFIG_DIR}/auth-configuration.conf"
+SSSD_CONF_FILE="/etc/sssd/sssd.conf"
+
+function trim_whitespace {
+  sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//'
+}
+
+function auth_type {
+  if [ -f ${AUTH_CONFIG_FILE} ]
+  then
+    CONFIG_AUTH_TYPE=$(grep '^type[[:space:]]*=.*$' ${AUTH_CONFIG_FILE} | cut -f2 -d= | trim_whitespace)
```

Comment:
This type of stuff really makes me want to have specs for something like this ...

Not sure if this would really work, but if we're going to have crystal on this pod anyway could we write this code up in a small crystal program so it would be easier to test? (This could also be a future thing, seeing as how we need to get auth in for the next release).